user-guide/Standard_Apps/SRM/SRM_migrating/SRM_migrating_booking_manager_full_configuration.md

```diff
@@ -0,0 +1,121 @@
+---
+uid: SRM_migrating_booking_manager_full_configuration
+---
+
+# Migrating a complete Booking Manager configuration
+
+The SRM framework allows you to export all artifacts of a Booking Manager application to facilitate deployment. You can then add the configuration to a Companion Files repository so it can be included in an install package, as detailed below.
+
+This functionality is available since version 1.2.24 of the SRM framework.
+
+Consider deploying the *Ziine SatUplink* booking application.
+
+## Exporting the configuration
+
+To export the configuration related to a specific Booking Manager application and its virtual platform, use the *SRM_ExportFullConfiguration* Automation script, which is included in the SRM package.
+
+1. In the Automation module, select the *SRM_ExportFullConfiguration* script and click *Execute*.
+
+1. As the input data for the script, specify a JSON object that defines which Booking Manager application should be exported.
+
+   For example:
+
+   ```json
+   { "BookingManagerName" : "Ziine SatUplink" }
+   ```
+
+1. Click *Execute now* to start the script.
+
+When the configuration has been exported, it will be available as a zip file in the *Skyline Booking Manager* > *Full Configurations* documents folder. The file name consists of the specified Booking Manager and the export time, e.g. "Ziine SatUplink_\`export time\`".
```

Comment:
Could you change the example to an exact example of the file name? That way people will be able to see the exact format of the name.